code/datums/mutations/radioactive.dm

```diff
@@ -9,8 +9,8 @@
 	power_coeff = 1
 
 
-/datum/mutation/human/radioactive/on_life()
-	radiation_pulse(owner, 20 * GET_MUTATION_POWER(src))
+/datum/mutation/human/radioactive/on_life(delta_time, times_fired)
+	radiation_pulse(owner, 10 * GET_MUTATION_POWER(src) * delta_time)
```

Comment:
I think that's basically just contamination? I'm not all too familiar with the effects of radiation and a quick skim doesn't show anything that would be a major issue. I guess in theory if something managed to hold SSmobs up for long enough this could generate a pulse large enough to hang SSmobs up for longer which would cause an even stronger pulse etc, etc. But, that's for when I get around to making SSmobs scale with lag.

Reply:
Looking at radiation code I don't think this will cause any major issues. I'll fix it if it does.

Reply:
Anturk's right, the strength of the radiation pulse is directly effects how far it can travel, and how much it can contaminate.
Converting it over like this will cause issues later if the wait of Life is ever raised, or if you implement that dynamic scaling idea of yours. It might just be best to leave it alone honestly.

Reply:
That's mostly a failure of how radiation works, but I see what you mean. I'll look into it.

Reply:
Have you addressed this?

Reply:
Even if dynamic delta time was implemented it would take SSmobs locking up for over 35 seconds for this to start contaminating things. I really don't think it will be a problem. Or at least not without much more pressing issues already present.